### ISSUE TITLE
Default postion of the map

### ISSUE THREAD

**Issue by dperrymorrow** [todgru/bus#4]
Hey instead of default zoom to Portland, the page could ask for your location and then let that be the default zoom for the map with a pin for user's current location.

Example code for that here...
http://toddmotto.com/using-html5-geolocation-to-show-current-location-with-google-maps-api/

**Comment by todgru** [comment]
:+1:

**Comment by dperrymorrow** [comment]
https://chadkillingsworth.github.io/geolocation-marker/
giving this lib a go

**Comment by dperrymorrow** [comment]
https://live-bus.herokuapp.com/
you can see it working here on this heroku instance i threw up today. The commit is added to the other pull as its on top of that js restructure

**Comment by todgru** [comment]
The last commit, 1764be03dc6654d61b42740b4a7f90bfb65e0bb7, to the js re org branch doesn't work in mobile, chrome or safari. 

Desktop Firefox, Chrome works as expected.

Let's pull this last commit out of https://github.com/todgru/bus/pull/2, I'd like to get that pr merged.

**Comment by dperrymorrow** [comment]
ok sounds good will do, sorry didnt test mobile, i think mobile has a few other problems as well. i think there is some runaway js on mobile. ill look into that as mentioned in #7

**Comment by todgru** [comment]
deployed to https://todgru.com/bus/

**Comment by todgru** [comment]
Interesting, but "myposition" shows up as a red marker, not the pulsing orb. (OSX Chrome and Firefox).

![screen shot 2015-04-30 at 5 47 39 pm](https://cloud.githubusercontent.com/assets/1738885/7425176/05950e0c-ef61-11e4-8e8d-d6835e6a34f7.png)

Hmm, this is crashing in Chrome on iOS.

**Comment by dperrymorrow** [comment]
yeah just using a regular marker ATM, https://github.com/todgru/bus/blob/master/public/js/map.js#L47 could be changed to anything though really. 

re crashing in chrome IOS, if you comment out the polling does it fix? 
https://github.com/todgru/bus/blob/master/public/js/map.js#L119

**Comment by todgru** [comment]
My recent changes seemed to have inadvertently fixed the crashing https://github.com/todgru/bus/commit/958a2cc1a2546c526f05aa16b200b29e6d802ca0 I'm not sure why.

Working great in dektop and ios chrome and ios safari.
